tools/go-cli/build.gradle

```diff
@@ -54,15 +19,16 @@ task distBinary(dependsOn: [removeBinary, distDocker]) << {
     run(["mkdir", "${projectDir}/../../bin/go-cli/"])
     run(dockerBinary + ["run", "--name", dockerContainerName, dockerImageName + ":" + dockerImageTag])
 
-    // Copy all Go binaries outside of Docker into openwhisk go-cli/bin/ folder
+    // Copy all Go binaries outside of Docker into openwhisk go-cli/build/ folder
     run(dockerBinary + ["cp", dockerContainerName +
```

Comment:
Quite right.  The comment did not keep up with the changes.  Copy destination is openwhisk/bin/go-cli